modules/troubleshooting-enabling-kdump.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operating-system-issues.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-kdump"]
+= Enabling kdump
+
+{op-system} ships with the `kexec-tools` package, but manual configuration is required to enable the `kdump` service.
+
+.Procedure
+
+Perform the following steps to enable kdump on {op-system}.
+
+. To reserve memory for the crash kernel during the first kernel booting, provide kernel arguments by entering the following command:
++
+[source,terminal]
+----
+# rpm-ostree kargs --append='crashkernel=256M'
+----
+
+. Optional: To write the crash dump over the network or to some other location, rather than to the default local `/var/crash` location, edit the `/etc/kdump.conf` configuration file.
++
+[NOTE]
+====
+Network dumps are required when using LUKS. Kdump does not support local crash dumps on LUKS-encrypted devices.
```

Comment:
`kdump` has a capitalized variant? Isn't it just _kdump_?

Reply:
IMO, the use of backticks here would be appropriate to start the sentence with `kdump`

The guidance for object references is going to apply to 99% of OCP, but RHCOS is a snowflake; my argument is that using backticks around a command name is appropriate when talking about a component of RHCOS or the operating system.

Reply:
That makes sense, thanks for explaining that

Reply:
@jldohmann, so in this case, I'd probably use backticks or use it as lowercase. Thanks!

Reply:
@jldohmann, how about

> If your node uses LUKS-encrypted devices, you must use network dumps as kdump does not support saving crash dumps to LUKS-encrypted devices.

Reply:
@jboxman-rh thank you! Updated